doc/radosgw/multisite.rst

```diff
@@ -1283,20 +1283,21 @@ Finally, update the period.
 Zones
 -----
 
-Ceph Object Gateway supports the notion of zones. A zone defines a
-logical group consisting of one or more Ceph Object Gateway instances.
+A zone defines a logical group that consists of one or more Ceph Object Gateway
+instances. Ceph Object Gateway supports zones.
```

Comment:
The most important property of a zone, namely that all RGWs in the zone look into the same pools in the same cluster, is not mentioned.

Reply:
I will add this tomorrow, when it is not the middle of the night. It's good to see you, @patrakov.

Reply:
@patrakov: Can you be more specific than "look into the same pools in the same cluster"? I'm seeking a completely unambiguous sentence. Should we say that all RGWs in a given zone target a given set of pools within a given cluster?